templates/stream.app.navigation.php

```diff
@@ -55,7 +55,7 @@
 			</span>
 			<div id="activity-personal-settings-link">
 				<a href="<?php p($_['personalSettingsLink']); ?>">
-					<span class="no-icon"><?php p($l->t('Personal Activity Settings')); ?></span>
+					<span class="no-icon"><?php p($l->t('Personal activity settings')); ?></span>
```

Comment:
Maybe we should rename it to Personal notification settings on NC23 and above?

Reply:
Very good idea 👍. More suited to the type of setup. I changed 😉.